cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -75,6 +75,7 @@
       cronjob at:'5 16 * * *', do:deploy_dir('bin', 'cron', 'calculate_workshop_survey_results')
       cronjob at:'20 6 * * 6', do:deploy_dir('bin', 'cron', 'dms_tasks')
       cronjob at:'30 14 * * 1-5', do:deploy_dir('bin', 'cron', 'deploy_to_test')
+      cronjob at:'10 22 * * *', do:deploy_dir('bin', 'cron', 'build_contact_rollups')
```

Comment:
This runs in the afternoon PST (contrary to the PR summary), as the time is specified in UTC. My assumption is you want 10:10 PM PST, in which case this should be `'10 5 * * *'`.